app/README.md

```diff
@@ -37,3 +42,60 @@ npm run start:app
 ```
 
 You can open the Measure Repository Service frontend application by navigating to [http://localhost:3001](http://localhost:3001) in your browser.
+
+
+# Frontend Features
+
+## Navigation
+
+The application is divided into two main sets of functionality, _Repository_ and _Authoring_, both of which are accessible from tabs at the top of the page.
+
+<img src="./static/tabs.png" alt="Screenshot of tabs" width="200"/>
+
+The Repository tab provides functionality for access and interacting with measure artifacts that are in the FHIR Measure Repository Service. The Authoring tab focuses on measure artifacts that are in the process of being drafted. Artifacts can be copied or moved between these two spaces through _draft_ and _release_ actions, described in the [Viewing](#viewing) and [Editing](#editing) sections.
+
+Within these tabs, the left panel navigation allows for selecting artifact types or searching (Repository only) for artifacts so that they can be browsed for viewing or actions.
+
+<img src="./static/navigation.png" alt="Screenshot of navigation" width="250"/>
+
+## Landing Page
+The landing page can be accessed on first load or by clicking the Measure Repository home button on the top left. This page links to resources to learn more about the Measure Repository and shows existing FHIR service capabilities.
+
+## Repository Tab
+The Repository tab displays sets of artifacts from the FHIR Measure Repository Service. Options allow for viewing resource details or reviewing the resource.
+
+<img src="./static/repository-options.png" alt="Screenshot of repository options" width="75"/>
+
+### Viewing
+-	Measure: Resource details include options to see the JSON, Data Requirements, or Dependencies.        
+-	Library: Resource details include options to see the JSON or Narrative.  
+-	Drafting: Both resource views allow the user to create a draft from the artifact. This action copies the existing artifact and any children it owns to the draft Authoring space with an incremented version to logically differentiate it from the original artifact.
```

Comment:
```suggestion
-	Drafting: Both resource views allow the user to create a draft from the artifact. This action copies the existing artifact and any children it owns to the draft Authoring space with an incremented version and draft status to logically differentiate it from the original artifact.
```
Or something along those lines if you think that's necessary

Reply:
```suggestion
-	Drafting: Both resource views allow the user to create a draft from the artifact. This action copies the existing artifact and any children it owns to the draft Authoring space with a draft status and an incremented version to logically differentiate it from the original artifact. See the [draft operation specification](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#draft).
```